server/wfss-pointid-api/wfss-pointid-api-rest-endpoints/src/main/resources/application-secrets.properties

```diff
@@ -1,5 +1,9 @@
 # Point ID secrets
 weather.user=#{WEATHER_USER}
 weather.password=#{WEATHER_PASSWORD}
+<<<<<<< HEAD
```

Comment:
It looks like the <<<<< separators from Git are in the file. We want to remove these and make sure we don't have any duplicated content. These indicate conflicts and shouldn't be merged as changes

Reply:
```
<<<<<<< HEAD
webade.oauth2.client.secret=#{POINTID_WEBADE_OAUTH2_CLIENT_SECRET}
=======
webade.oauth2.client.secret=#{WEBADE_OAUTH2_CLIENT_SECRET}
>>>>>>> d3f08d482eaafd2926976e2fa9258aa7ad988cc9
```

should be 

```
webade.oauth2.client.secret=#{WEBADE_OAUTH2_CLIENT_SECRET}
```

I think?